package.json

```diff
@@ -31,13 +31,17 @@
     "@storybook/react": "^8.1.5",
     "@storybook/react-webpack5": "^8.1.5",
     "@storybook/test": "^8.1.5",
+    "@svgr/webpack": "^6.1.2",
     "@types/css-modules": "^1.0.5",
     "@types/detect-port": "^1.3.5",
     "@types/diff": "^5.2.1",
     "@types/ejs": "^3.1.5",
     "@types/normalize-package-data": "^2.4.4",
     "@types/pretty-hrtime": "^1.0.3",
+    "@types/react": "^18.3.3",
+    "@types/react-i18next": "^8.1.0",
     "css-loader": "5.2.6",
+    "file-loader": "^6.2.0",
```

Comment:
what is the use case of file loader ?

Reply:
It is used to handle all media imports using webpack, if i remove this, i will have to change the structure of the project